src/dawn/CodeGen/CXXNaive/CXXNaiveCodeGen.cpp

```diff
@@ -548,6 +548,7 @@ std::unique_ptr<TranslationUnit> CXXNaiveCodeGen::generateCode() {
   };
 
   ppDefines.push_back(makeDefine("GRIDTOOLS_CLANG_GENERATED", 1));
+  ppDefines.push_back("#define GRIDTOOLS_CLANG_BACKEND_T CXXNAIVE");
```

Comment:
why do we need this?

Reply:
Issue with normal functions in benchmarking

Reply:
ok